taxi/views.py

```diff
@@ -34,12 +35,51 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer_list")
```

Comment:
The `success_url` in `ManufacturerCreateView` should match the URL pattern name defined in `urls.py`. It seems like it should be `taxi:manufacturer-list` instead of `taxi:manufacturer_list`.